tasks/constants.py

```diff
@@ -0,0 +1,28 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from __future__ import print_function, unicode_literals
+
+import os
+
+
+ROOT = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Since we're in a directory now, this should be `ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))`.